<launch>
   <rosparam file="$(find nao_teleop)/config/ps3controller.yaml" command="load" ns="teleop_nao_joy" />
</launch>
